pkg/postgres/time.go

```diff
@@ -46,5 +46,6 @@ func (ae *EasternTime) Scan(value interface{}) error {
 
 //Value implements the db/sql.Valuer interface
 func (ae EasternTime) Value() (driver.Value, error) {
+	ae = EasternTime(time.Time(ae).In(EasternTimeZone))
```

Comment:
This wasn't a problem because all the `time.Time`'s we _had_ been passing into SQL queries were already in Eastern. Adding this line doesn't change anything but helps make sure that `GetRecentlyActiveRuns` works properly no matter what `time.Location` the `touchThreshold` argument is in.